pkgs/applications/networking/cisco-packet-tracer/8.nix

```diff
@@ -21,106 +22,90 @@
 , makeWrapper
 , nspr
 , nss
+, qt5
 , requireFile
 , xorg
 }:
 
 let
-  version = "8.2.0";
-
-  ptFiles = stdenv.mkDerivation {
-    name = "PacketTracer8Drv";
-    inherit version;
-
-    dontUnpack = true;
-    src = requireFile {
-      name = "CiscoPacketTracer_${builtins.replaceStrings ["."] [""] version}_Ubuntu_64bit.deb";
-      sha256 = "1b19885d59f6130ee55414fb02e211a1773460689db38bfd1ac7f0d45117ed16";
-      url = "https://www.netacad.com";
-    };
-
-    nativeBuildInputs = [
-      alsa-lib
-      autoPatchelfHook
-      dbus
-      dpkg
-      expat
-      fontconfig
-      glib
-      libdrm
-      libglvnd
-      libpulseaudio
-      libudev0-shim
-      libxkbcommon
-      libxml2
-      libxslt
-      makeWrapper
-      nspr
-      nss
-    ] ++ (with xorg; [
-      libICE
-      libSM
-      libX11
-      libxcb
-      libXcomposite
-      libXcursor
-      libXdamage
-      libXext
-      libXfixes
-      libXi
-      libXrandr
-      libXrender
-      libXScrnSaver
-      libXtst
-      xcbutilimage
-      xcbutilkeysyms
-      xcbutilrenderutil
-      xcbutilwm
-    ]);
-
-    installPhase = ''
-      dpkg-deb -x $src $out
-      chmod 755 "$out"
-      makeWrapper "$out/opt/pt/bin/PacketTracer" "$out/bin/packettracer" \
-        --prefix LD_LIBRARY_PATH : "$out/opt/pt/bin"
-
-      # Keep source archive cached, to avoid re-downloading
-      ln -s $src $out/usr/share/
-    '';
+  hashes = {
+    "8.2.0" = "1b19885d59f6130ee55414fb02e211a1773460689db38bfd1ac7f0d45117ed16";
+    "8.2.1" = "1fh79r4fnh9gjxjh39gcp4j7npgs5hh3qhrhx74x8x546an3i0s2";
   };
+in
 
-  desktopItem = makeDesktopItem {
-    name = "cisco-pt8.desktop";
-    desktopName = "Cisco Packet Tracer 8";
-    icon = "${ptFiles}/opt/pt/art/app.png";
-    exec = "packettracer8 %f";
-    mimeTypes = [ "application/x-pkt" "application/x-pka" "application/x-pkz" ];
-  };
+stdenvNoCC.mkDerivation rec {
+  pname = "ciscoPacketTracer8";
 
-  fhs = buildFHSEnv {
-    name = "packettracer8";
-    runScript = "${ptFiles}/bin/packettracer";
-    targetPkgs = pkgs: [ libudev0-shim ];
+  version = "8.2.1";
```

Comment:
I understand and largely aggree with the hashes dictionary.
However, I would vote for parameterize this.

But not today.